README.md

```diff
@@ -36,10 +36,13 @@ The last one can be run by:
   - users in the `quizzo-manage-secrets` group
   - programs on ec2 instances which have been started with the `quizzo-secrets` IAM profile
 
-To start an instance with the quizzo-secrets IAM profile from the CLI:
+To start an EC2 instance with the quizzo-secrets IAM profile from the CLI:
 
   `aws ec2 run-instances ...--iam-instance-profile Name=quizzo-secrets`
 
+To start an ECS cluster with the quizzo IAM profile, select quizzo-secrets-instances from the
+Container Instance IAM Role selection on the Create Cluster screen.
```

Comment:
@themoxman would be a good one to pair with for ECS. AFAIK he only ever did it through the AWS console, but he would understand the flow well, I think.